simSPI/transfer.py

```diff
@@ -36,12 +41,14 @@ def ctf_freqs(n_pixels, psize=1.0, dim=2):
     """
     if dim == 1:
         freq_pix_1d = np.arange(0, 0.5, 1 / n_pixels)
-        freq_1d = freq_pix_1d * psize
+        freq_pix_1d_safe = freq_pix_1d[: n_pixels // 2]
```

Comment:
This looks like it's solving a different error: that line was forgotten initially, correct?

Reply:
Normally freq_pix_1d was the right size of `n_pixels // 2`, but then for size 98 it gets this extra element because of how np.array treats things (closed vs open intervals I suppose). So I just enforce that it doesn't have this extra point. The intervals are all good, before it just sometimes had an extra end point.